sdk/src/frame_impl.cpp

```diff
@@ -130,8 +130,9 @@ aditof::Status FrameImpl::getData(aditof::FrameDataType dataType,
 }
 
 void FrameImpl::allocFrameData(const aditof::FrameDetails &details) {
-    m_fullData = new uint16_t[details.width * details.height * 2 +
-                              details.rgbWidth * details.rgbHeight];
+    //removed the usage of rgbWidth/height to avoid app crash when running on target
```

Comment:
What is this about?

Reply:
Fixed. Found another way to disable the allocation